Libraries/LibGUI/GFilePicker.h

```diff
@@ -7,22 +7,41 @@ class GLabel;
 
 class GFilePicker final : public GDialog {
 public:
-    GFilePicker(const StringView& path = "/", CObject* parent = nullptr);
+    enum class Mode {
+        Open,
+        Save
+    };
+
+    static String get_open_filepath();
+    static String get_save_filepath();
+
+    GFilePicker(Mode type = Mode::Open, const StringView& path = "/", CObject* parent = nullptr);
     virtual ~GFilePicker() override;
 
     FileSystemPath selected_file() const { return m_selected_file; }
 
-    virtual const char* class_name() const override { return "GFilePicker"; }
```

Comment:
Please don't remove this. All `CObject` derived classes should implement `class_name()` so they can be identified at runtime.